docs/source/developing/ci.adoc

```diff
@@ -10,21 +10,51 @@ toc::[]
 [[ci-overview]]
 == Overview
 
-The following sections describes the various CI setups Minishift utilizes.
+Minishift uses several CI build servers such as Centos, AppVeyor, Circle and Travis. These servers build incoming pull requests, pass it through various test parameters and provide the status of the builds to ensure continuous integration and deployment. Except for Travis CI, they also archive build artifacts of successful builds.
```

Comment:
Agreed, will remove such as and change appropriately.

"pass it through various test parameters" - what's that supposed to mean?
> The Circle CI check at the bottom of all PRs says 'ci/circleci — Your tests passed on CircleCI!'  so based on this and my discussions with @praveenkumar afaiu, these servers build the PRs after passing them through various checks. Can you please help me understand what 'Your tests passed on CircleCI!' mean.

"ensure continuous integration and deployment" - not sure about this. We for sure don't do "continuous deployment".
> well both Circle and AppVeyor claim to do both, and that's where I got it from. I will change it to ensure continuous integration. I hope that is alright, considering they are called CI servers I am guessing that is what we use them for.